extensions/adobe/experience/adcloud/searchads/transactionproperties.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/adcloud/searchads/transactionproperties",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Advertising Transaction Property",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "meta:auditable": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Advertising Transaction schemas includes metadata that describes a transaction property setup by Advertiser",
+  "definitions": {
+    "searchads-transaction": {
+      "properties": {
+        "searchads:userId": {
```

Comment:
I would prefer to call this as a clientId and explicitly mention Ad Cloud Client Id in the descriptive text